impeller/playground/backend/vulkan/playground_impl_vk.cc

```diff
@@ -60,16 +60,7 @@ void PlaygroundImplVK::DestroyWindowHandle(WindowHandle handle) {
 
 PlaygroundImplVK::PlaygroundImplVK(PlaygroundSwitches switches)
```

Comment:
Ideally this would be a static function that returns `StatusOr<std::unique_ptr<PlaygroundImplVK>>` instead of short circuiting a constructor, leaving a half initialized object.

https://google.github.io/styleguide/cppguide.html#Doing_Work_in_Constructors

Reply:
I think we can just lift the check out of the constructor, since its a static method. I made this an FML_CHECK

Reply:
Done